README.md

```diff
@@ -26,33 +27,44 @@ Each versioned migration must be assigned a unique version.
 - 2013.01.15.11.35.56
 
 ### Script Config Files
-Similar to [Flyway script config files](https://documentation.red-gate.com/flyway/flyway-cli-and-api/configuration/script-config-files) it's possible to configure migrations on a per-script basis.  
+
+Similar to [Flyway script config files](https://documentation.red-gate.com/flyway/flyway-cli-and-api/configuration/script-config-files) it's possible to configure migrations on a per-script basis.
 
 This is achieved by creating a script configuration file in the same folder as the migration.  
-The script configuration file name must match the migration file name, with the `.conf` suffix added.  
+The script configuration file name must match the migration file name, with the `.conf` suffix added.
 
 For example a migration file `db/V1__my_script.sql` would have a script configuration file `db/V1__my_script.sql.conf`.
 
 #### Structure
 
 Script config files have the following structure:
+
 ```
 key=value
 ```
 
 #### Reference
 
 - **executeInTransaction**  
-  Manually determine whether or not to execute this migration in a transaction. 
+  Manually determine whether or not to execute this migration in a transaction.
 
   This is useful where certain statements can only execute outside a transaction (like `CREATE INDEX CONCURRENTLY` etc.)  
   Example:
+
   ```
   executeInTransaction=false
   ```
 
 ## Usage example
 
+For usage via command line see [command-line](#command-line) section.
+
+In a sbt project dumbo can be added like:
+
+```
+libraryDependencies += "dev.rolang" %% "dumbo" % "0.0.7"
+```
+
```

Comment:
I would also mention in documentation about snapshot releases that you need to configure `resolvers`, e.g.

```scala
ThisBuild / resolvers +=
  "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
```

Reply:
right, will add, thanks for the hint :+1:

Reply:
updated

Reply:
thanks